core/controllers/reader.py

```diff
@@ -108,7 +110,7 @@ def classify_string_classifier_rule(state, normalized_answer):
         labels = sc.predict([normalized_answer])
         predicted_label = labels[0]
         if (predicted_label !=
-                lda_string_classifier.LDAStringClassifier.DEFAULT_LABEL):
+                sc.DEFAULT_LABEL):
```

Comment:
I was going to ask you this, but then I thought one cannot keep putting all the constants in feconf, so I let it remain in StringlClassifier. I'll update it.